pyEpiabm/pyEpiabm/sweep/assign_household_ages_sweep.py

```diff
@@ -0,0 +1,394 @@
+#
+# Sweep to assign ages to people in households
+#
+
+import random
+from scipy.stats import poisson
+import numpy as np
+
+from pyEpiabm.core import Person
+from pyEpiabm.core import Parameters
```

Comment:
```suggestion
from pyEpiabm import Person, Parameters
```
Does this work? Closer to the import style we try to achieve if it is possible